contrib/gorilla/mux/mux.go

```diff
@@ -108,6 +108,7 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	spanopts = append(spanopts, httptraceinternal.HeaderTagsFromRequest(req, r.config.headerTags))
 	resource := r.config.resourceNamer(r, req)
 	httptrace.TraceAndServe(r.Router, w, req, &httptrace.ServeConfig{
+		Framework:     "github.com/gorilla/mux",
```

Comment:
@RomainMuller Do all HTTP contribs need to register this `Framework` field? It's to add this to contribution guidelines.

Reply:
AppSec basically uses this to tag telemetry when we are unable to obtain some information... So this is optional, but I reckon helps if present (that might be useful for APM telemetry, too, in the future?)